### PR TITLE
Updated the README.md file

### LINKED ISSUE #1: Update Readme.md : Add some screenshot of website
You can add some screenshots of the website for better reach and understanding.

1. Main page Page
2. Shopping cart Page
3. Address Form Page

### PR DESCRIPTION
**Pull request by vishwast03**
@kavania2002 
I have updated the README.md file as you have described. Please take it into your consideration. Suggestions are welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 ![MobiMart](https://socialify.git.ci/kavania2002/MobiMart/image?description=1&descriptionEditable=MobiMart%20is%20a%20mobile%20shopping%20website.%20Very%20useful%20for%20Django%20beginner.&font=Bitter&forks=1&issues=1&language=1&owner=1&pattern=Plus&pulls=1&stargazers=1&theme=Light)
 <br>
 
+# MobiMart
+<br>
+
+![Main Page ss1](https://user-images.githubusercontent.com/78464111/107855526-d4186000-6e48-11eb-9364-35e753427f8e.png)
+<br>
+![Shopping Cart ss](https://user-images.githubusercontent.com/78464111/107855564-32454300-6e49-11eb-838a-0bbf5c13146d.png)
```
Comment (kavania2002) [inline README.md]: Why the tab of logout and login showing simultaneously. See that it is running properly. You can add screenshot from the website given.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
 ![MobiMart](https://socialify.git.ci/kavania2002/MobiMart/image?description=1&descriptionEditable=MobiMart%20is%20a%20mobile%20shopping%20website.%20Very%20useful%20for%20Django%20beginner.&font=Bitter&forks=1&issues=1&language=1&owner=1&pattern=Plus&pulls=1&stargazers=1&theme=Light)
 <br>
 
+# MobiMart
+<br>
+
+![Main Page ss1](https://user-images.githubusercontent.com/78464111/107855526-d4186000-6e48-11eb-9364-35e753427f8e.png)
```
Comment (kavania2002) [inline README.md]: Add some more images of index page. The ones containing mobile phones.

**Review by kavania2002 — COMMENTED** [review]
Do join the discord server.

File: README.md (hunk 1 above)
Comment (vishwast03) [reply README.md]: Ok i will check that

File: README.md (hunk 2 above)
Comment (vishwast03) [reply README.md]: Ok
Thanks for suggestions

**Comment by vishwast03** [comment]
@kavania2002 
I have again updated the README.md file. Please check it.

**Comment by kavania2002** [comment]
Good job.

**Comment by kavania2002** [comment]
#1 done by @vishwast03

**Comment by vishwast03** [comment]
@kavania2002 
Thanks

**Pull request merged**